### PR TITLE
Added documentation for the ELV PCA 301 smart emeter switch

### PR DESCRIPTION
**Pull request by majuss**
**Description:**
Adds the documentation for the ELV PCA 301 smart emeter switch running at 868 MHz.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#23300

## Checklist:

- [ ] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [ ] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/switch.pca.markdown
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]:
```suggestion
  description: "The path to you serial tty. Get it via: `ls /dev/tty*`."
```

**Comment by majuss** [comment]
Had in typo in there, now it should work.

File: source/_components/switch.pca.markdown
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]:
```suggestion
ha_iot_class: Local Polling
```

File: source/_components/switch.pca.markdown
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]:
```suggestion
ha_release: 0.91
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_components/switch.pca.markdown
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown:31]: I see that there is a config variable for `name`, you can add this here.

File: source/_components/switch.pca.markdown
Comment (klaasnicolaas) [inline source/_components/switch.pca.markdown]: One whitespace to much 😉

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
After this changes, we can approve it 🎉

**Review by klaasnicolaas — DISMISSED** [review]
✅ Can be merged after parent PR got merged 👍

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @majuss! 👍

**Comment by majuss** [comment]
It's always a pleasure to work with you guys and to expand the supported platforms! :)

**Comment by majuss** [comment]
Hm something is off, maybe some pca/elv mixup, I can't reach the documentation.

**Comment by klaasnicolaas** [comment]
https://www.home-assistant.io/components/switch.pca/

Maybe it was better if the platform was called `elf` and the linke ended with /elv/

**Pull request merged**
